### DISCUSSION TITLE
Is there a way to search across a solution?

### DISCUSSION THREAD

**Discussion by dandax** [OmniSharp/omnisharp-vscode#7247] [General]
I've got a solution that references other projects from outside the current solution directory. Is there a way to search those files as well? For example, can the C# Dev Kit determine the folders/files in other projects and include them in the global search tool?

**Comment by dibarbet** [comment]
@dandax if the projects are part of the `.sln` file loaded by the extension, C#/C# Dev Kit items in those projects should show up in the various navigation features (find references, workspace symbol search, etc).  

For the regular find in files VSCode feature, I believe it will only search the opened workspace folders.  You can add multiple folders to the workspace (see https://code.visualstudio.com/docs/editor/multi-root-workspaces), but note that currently C#/C# Dev Kit only support loading one solution.
